meta-sokol-flex-staging/files/relocate_sdk.sh

```diff
@@ -43,7 +43,7 @@ if [ -e "$native_sysroot/lib" ]; then
         exit 1
     fi
 
-    dl_path=$(find $native_sysroot/lib -name "ld-linux*")
+    dl_path=$(find $native_sysroot/lib -mindepth 1 -maxdepth 1 -name "ld-linux*" | head -n 1)
```

Comment:
Can't we simply do "ls $native_sysroot/lib/ld-linux* | head -n 1" instead?

Reply:
Based on Chris comment I am merging this change as Customer has tested this.